### DISCUSSION TITLE
Add a "Magic Wand" selection tool for Toonz Raster and Raster Layer.

### DISCUSSION THREAD

**Discussion by GeniyX** [opentoonz/opentoonz#6769] [Ideas]
Suggestion to add "Magic Wand" selection tool to OpenToonz. The work of the tool is supposed to be similar in functionality as in Krita, Photoshop or GIMP.
Tool documentation in Krita: https://docs.krita.org/en/reference_manual/tools/contiguous_select.html

Recently, one of my friend tried using OpenToonz for drawing/animation and was upset by the lack of such a basic and familiar tool. I think many artists will be happy to see this functionality in the program.

This tool seems most useful on a raster layer, since on a vector we, in principle, can already create a semblance of such a functionality by selecting individual lines.

**Comment by oppotaku** [comment]
This would be such a useful tool.
